Testreglar/1.1.1/Nett/nett-1.1.1a.json

```diff
@@ -137,7 +137,7 @@
-			"ht": "<p>Ikoner og grafiske elementer som er lagt til for å fremheve det visuelle, kan ofte oppfattes som unødvendig \"støy\" eller forstyrrende for skjermleserbrukere (<a href=\"https://medialt.no/bildebeskrivelser/1481.aspx\" target=\"_blank\" rel=\"noopener\">MediaLT</a>).</p>\n<p>Sjekk om bildet</p>\n<ul>\n<li>er ren dekorasjon eller</li>\n<li>brukes utelukkende til visuell formatering eller</li>\n<li>ikke presenteres for brukeren</li>\n</ul>\n<p><strong>Ren dekorasjon: </strong>Begrepet \"<a href=\"https://www.w3.org/Translations/WCAG21-no/#dfn-pure-decoration\" target=\"_blank\" rel=\"noopener\"><em>ren dekorasjon</em></a>\" er definert i WCAGs ordliste: \"<em>innhold som bare har et estetisk formål, som ikke inneholder informasjon eller funksjonalitet\". </em>Likevel er<em> </em>det en skjønnsmessig vurdering hvilke bilder det gjelder. Definisjonen peker på to tilfeller:</p>\n<ul>\n<li>bildet har kun et estetisk formål</li>\n<li>bildet inneholder ikke informasjon eller funksjonalitet.</li>\n</ul>\n<p><strong>Utelukkende visuell formattering</strong></p>\n<p>Eksempel på bilde som brukes utelukkende til visuell formattering er en bord eller en virvel som brukes til å skape en visuell effekt.</p>\n<p><strong>Ikke viser (presenteres) for brukeren</strong></p>\n<p>Bilde som ikke vises til brukeren er usynlige bilder, som for eksempel brukes for å spore bruksstatistikk på en nettside.</p>",
+			"ht": "<p>Ikoner og grafiske elementer som er lagt til for å fremheve det visuelle, kan ofte oppfattes som unødvendig \"støy\" eller forstyrrende for skjermleserbrukere (<a href=\"https://medialt.no/bildebeskrivelser/1481.aspx\" target=\"_blank\" rel=\"noopener\">MediaLT</a>).</p>\n<p>Sjekk om bildet</p>\n<ul>\n<li>er ren dekorasjon eller</li>\n<li>brukes utelukkende til visuell formatering eller</li>\n<li>ikke presenteres for brukeren</li>\n</ul>\n<p><strong>Ren dekorasjon: </strong>Begrepet \"<a href=\"https://www.w3.org/Translations/WCAG21-no/#dfn-pure-decoration\" target=\"_blank\" rel=\"noopener\"><em>ren dekorasjon</em></a>\" er definert i WCAGs ordliste: \"<em>innhold som bare har et estetisk formål, som ikke inneholder informasjon eller funksjonalitet\". </em>Likevel er<em> </em>det en skjønnsmessig vurdering hvilke bilder det gjelder. Definisjonen peker på to tilfeller:</p>\n<ul>\n<li>bildet har kun et estetisk formål</li>\n<li>bildet inneholder ikke informasjon eller funksjonalitet.</li>\n</ul>\n<p><strong>Merk:</strong> Der bildet ikke gir mer informasjon eller funksjonalitet enn løpende tekst som hører til bildet, er bildet vanligvis dekorativt.</p>\n<p><strong>Utelukkende visuell formattering</strong></p>\n<p>Eksempel på bilde som brukes utelukkende til visuell formattering er en bord eller en virvel som brukes til å skape en visuell effekt.</p>\n<p><strong>Ikke viser (presenteres) for brukeren</strong></p>\n<p>Bilde som ikke vises til brukeren er usynlige bilder, som for eksempel brukes for å spore bruksstatistikk på en nettside.</p>",
```

Comment:
Bruk samme tittel i uthevet som i kulepunkt. Bruk feks. "Ikke presenteres for brukeren" begge steder